### ISSUE TITLE
Body Element

### ISSUE THREAD

**Issue by X-Renegade-X** [Zefau/ioBroker.jarvis#877]
Hi,

wäre es nicht möglich die Bodys für die RGB Steuerung kleiner zu machen? bzw. bei hue sogar nur den Schieberegler für RGB einzusetzen. Aktuell nutze ich den hue Datenpunkt mit dem levelBody, um die Farbe einzustellen.

Wären auch 2 bodys möglich? Zum Beispiel Dimmen und Farbeinstellung (egal ob Farbtemperatur oder RGB) untereinander. So könnte man die Basics komplett ohne Popup steuern. gerade bei Sammelanteuerung wäre es somit auch leichter, denn dann muss man nur 2 Sammeldatenpunkte anlegen und kann das Popup unterdrücken.

**Comment by Zefau** [comment]
Sind mehrere Anfragen. Schau mal in der v3, was damit ggf. schon abgedeckt ist. Oder beziehst du dich bereits auf v3?

**Comment by X-Renegade-X** [comment]
Nein, bin aktuell auf 2.2.1. Will da erstmal alles stable bekommen und dann -> backup und auf v3.
